.github/main.workflow

```diff
@@ -40,10 +40,9 @@ action "tag images" {
 
 action "push images" {
   uses = "actions/docker/cli@master"
-  runs = "sh -c"
   env = {
     IMAGE_NAME = "gcr.io/kubernetes1-226021/capd-manager"
   }
-  args = "source $HOME/.profile && docker push $IMAGE_NAME:latest && docker push $IMAGE_NAME:$IMAGE_REF && docker push $IMAGE_NAME:$IMAGE_SHA && docker push $IMAGE_NAME:$IMAGE_VERSION"
+  args = ["source $HOME/.profile && docker push $IMAGE_NAME:latest && docker push $IMAGE_NAME:$IMAGE_REF && docker push $IMAGE_NAME:$IMAGE_SHA && docker push $IMAGE_NAME:$IMAGE_VERSION"]
```

Comment:
> If you provide args in a string, the string is split around whitespace.

https://developer.github.com/actions/managing-workflows/workflow-configuration-options/#actions-attributes

Do you think we'll have a better chance doing something like
```
runs = ["sh", "-c", "source $HOME/.profile && docker push $IMAGE_NAME:latest && docker push $IMAGE_NAME:$IMAGE_REF && docker push $IMAGE_NAME:$IMAGE_SHA && docker push $IMAGE_NAME:$IMAGE_VERSION"]
```

Reply:
i think i shouldn't have removed the runs 🤔 oh well. i'll try that out next

Reply:
one big problem is lack of actual logs. it just says "failed" at this point and doesn't tell me why